apps/developer-hub/vercel.json

```diff
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://openapi.vercel.sh/vercel.json",
+  "ignoreCommand": "../../vercel-ignore.sh",
```

Comment:
We recently got rid of the ignore command and are using a new vercel feature for this instead, see #2701 for details